pkg/comp-functions/functions/vshnpostgres/maintenance.go

```diff
@@ -53,18 +53,6 @@ var (
 				"get",
 			},
 		},
-		{
-			APIGroups: []string{
-				"vshn.appcat.vshn.io",
-			},
-			Resources: []string{
-				"vshnpostgresqls",
```

Comment:
this should be unnecessary as we don't expect vshnpostgresqls to be in the instance namespace, therefore it has been moved to the cluster role that is defined in the component